src/main/java/com/openlattice/chronicle/services/entitysets/EntitySetIdsService.java

```diff
@@ -309,9 +311,11 @@ public ChronicleCoreAppConfig getLegacyChronicleAppConfig( String participantESN
                 legacyEntitySetIds.get( HAS ),
                 optional,
                 legacyEntitySetIds.get( PARTICIPATED_IN ),
+                legacyEntitySetIds.get( MESSAGES ),
                 legacyEntitySetIds.get( METADATA ),
                 legacyEntitySetIds.get( PART_OF ),
                 legacyEntitySetIds.get( NOTIFICATION ),
+                legacyEntitySetIds.get( SENT_TO ),
```

Comment:
not necessary. Feature won't be available for legacy, so we shouldn't have to add new entity sets